### PR TITLE
(CTR) build fix.

### PR DESCRIPTION
**Pull request by aliaspider**

### CODE REVIEW

**Comment by twinaphex** [comment]
thanks for this,

File: menu/cbs/menu_cbs_select.c [hunk 1]
```diff
@@ -148,11 +148,13 @@ static int action_select_input_desc(const char *path, const char *label, unsigne
    return action_right_input_desc(type, label, true);
 }
 
+#ifdef HAVE_KEYMAPPER
```
Comment (RobLoach) [inline menu/cbs/menu_cbs_select.c]: Seem unrelated to the CTR fixes.

Comment (aliaspider) [reply menu/cbs/menu_cbs_select.c]: well it fixes targets with HAVE_KEYMAPPER=0. there aren't that many, although I'm not even sure if that build setting should be disabled in the first place for CTR:
